wgengine/router/callback.go

```diff
@@ -18,6 +18,13 @@ type CallbackRouter struct {
 	SetBoth  func(rcfg *Config, dcfg *dns.OSConfig) error
 	SplitDNS bool
 
+	// GetBaseConfigFn optionally specifies a function to return the current DNS
+	// config in response to GetBaseConfig().
```

Comment:
drop parens unless you're referring to the result of the call

Reply:
Done